formats/publication/publisher/details.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "body",
+    "documents",
+    "first_public_at",
+    "government",
+    "political"
+  ],
+  "properties": {
+    "body": {
+      "type": "string"
+    },
+    "documents": {
+      "type": "array",
+      "items": {
+        "type": "string"
```

Comment:
I don't understand your comment.

They are responsible for this bit in the frontend:
<img width="1082" alt="screen shot 2016-05-06 at 13 41 08" src="https://cloud.githubusercontent.com/assets/319055/15073224/43bcfd6c-1390-11e6-937d-c7e9be95c028.png">

Details about this in this commit message, with links to decisions: 08d08f29b34275049b096e0239603308a30ff5bc